Makefile.am

```diff
@@ -225,3 +225,28 @@ EXTRA_DIST += \
   config/stl_hash.m4 \
   mac_build.sh \
   ola.spec
+
+# -----------------------------------------------------------------------------
+
+# Create a linter target
+.PHONY : lint
+lint: flake8 cpplint
+
+# flake8 linter
+.PHONY : flake8
+flake8: Makefile.am
+if FOUND_FLAKE8
+	flake8
+else
+	$(error flake8 not found. Install flake8 and re-run configure.)
+endif
+
+# cpplint linter
+CPP_LINT_URL = "https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py"
+.PHONY : cpplint
+cpplint: Makefile.am cpplint.py
+	./run_cpplint.sh
+
+cpplint.py:
+	wget -O cpplint.py $(CPP_LINT_URL)
+	chmod u+x cpplint.py
```

Comment:
I'd remove this, or at least not make it required as part of cpplint phony. While we need it on Travis to run the check, as a developer I'd probably have the script in my path stored once (and indeed do), rather than once per OLA checkout. I also may have installed it via other means.

Reply:
Done.